compiler/ast/dag/op.go

```diff
@@ -58,6 +58,14 @@ type (
 		RightKey Expr         `json:"right_key"`
 		Args     []Assignment `json:"args"`
 	}
+	Load struct {
+		Kind    string `json:"kind" unpack:""`
+		Pool    string `json:"pool"`
+		Author  string `json:"author"`
+		Message string `json:"message"`
+		Meta    string `json:"meta"`
+		Branch  string `json:"branch"`
```

Comment:
Put Branch after Pool since they're coupled.
```suggestion
		Kind    string `json:"kind" unpack:""`
		Pool    string `json:"pool"`
		Branch  string `json:"branch"`
		Author  string `json:"author"`
		Message string `json:"message"`
		Meta    string `json:"meta"`
```